doc/source/conf.py

```diff
@@ -36,6 +36,9 @@
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
     'sphinx.ext.mathjax',
+    'sphinxcontrib.fulltoc',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.napoleon',
```

Comment:
What is this one for? Just curious.

Reply:
I introduced that for the `message_ix` doc pages to fix a million (erroneous) warning messages of not finding dedicated pages for each function. Copied it here when trying to get the `make html` work on my PC.

Reply:
Gotcha, cool